tetris.py

```diff
@@ -181,14 +197,41 @@ def new_block(self):
         self.current_block_center = (int(game_width / 2), -2)
         self.block_letter = self.small_board.block_letter
         self.small_board.block_letter = random.choice(block_letters)
+        self.small_board.resize_to_fit()
         self.small_board.draw_block()
         self.rotate_counter = 0
         self.fast_down = False
 
+    def resize_to_fit(self):
+        if self.block_letter == 'I':
+            self.canvas.config(width=square_size * 4, height=square_size * 1)
+            self.canvas.pack(pady=square_size/2+5)
+            self.current_block_center = (1, 0)
+        else:
+            self.current_block_center = (1, 1)
+        if self.block_letter == 'L':
```

Comment:
A blank line above `if self.block_letter == 'L'` would help separate the first `if,else` from the rest

Reply:
Done